proto/api/v1/metadata_service.proto

```diff
@@ -90,6 +90,9 @@ message CentralServicesCapabilities {
 
     // Ability to configure and perform Central backups to Amazon S3 or Google Cloud Storage.
     CapabilityStatus central_can_use_cloud_backup_integrations = 2;
+
+    // Ability to accept resources created declaratively by user.
+    CapabilityStatus central_can_accept_user_declarative_resources = 3;
```

Comment:
@ivan-degtiarenko do you want to add corresponding property to frontend service file?
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/services/MetadataService.ts#L33-L39

Reply:
Thanks for the pointer @pedrottimark . I think we'll merge this PR into my PR as-is, make further backend edits there and address `MetadataService.ts` there also or as a separate PR.